docs/source/models.rst

```diff
@@ -124,28 +132,45 @@ class:
         (cm)", "type": "double"}, {"name": "petal length (cm)", "type": "double"}, {"name":
         "petal width (cm)", "type": "double"}]'
       outputs: '[{"type": "integer"}]'
+      
+Tensor-based Signature Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
add in "Only DL flavors support tensor-based signatures (i.e TensorFlow, Keras, PyTorch, Onnx, and Gluon)."